modules/_labs/select/react/lib/Select.tsx

```diff
@@ -488,7 +488,8 @@ class Select extends React.Component<SelectProps, SelectState> {
         case 'Enter':
           isShortcut = true;
           if (menuVisibility === 'closed' || menuVisibility === 'closing') {
-            this.toggleMenu(true);
+            // allow the click
+            isShortcut = false;
```

Comment:
Fixes this error: https://dashboard.cypress.io/projects/odida5/runs/2269/test-results/b8041582-848c-4085-ae92-85e6be37458e

> `event.preventDefault()` was called on this element or a parent element and no additional functionality' simulated default actions. This prevents keyboard accessibility on <button> elements. Read more at https://github.com/cypress-io/cypress/issues/311. Use `{force: true}` to disable this check